doc/Makefile.am

```diff
@@ -2,8 +2,6 @@
 # Makefile.am for .../agent-ldap/doc
 #
 
-SUBDIRS = autodocs
```

Comment:
Yes, this is a leaf(ish) package so most people don't care about the api docs, but those who do should have an easy way to do it.

So, both approaches that you suggested are wrong :)

You should keep the SUBDIRS, but instead of including `autodocs-cc.ami` use `autodocs-cc-off.ami`